index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <script src="https://kit.fontawesome.com/a012cea4ea.js" crossorigin="anonymous"></script>
+    <link href="style.css" rel="stylesheet"/>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+      <link rel="stylesheet" href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+</head>
+<body>
+   <header class="main-header">
+    <div class="header-container">   
+    <a href="" class="logo text-color">Solomon</a>
+    <nav>
+        <span class="material-symbols-outlined text-color">
+            menu
+            </span>
+    </nav>
+    </div> 
+   </header>
+   <section class="section-1">
+       <div class="heading-1">
+            <h1 class="heading-1 margin-24">Hi there,</h1>
+            <h1 class="heading-1 margin-24">I'm Solomon</h1>
+       </div>
+       <h2 class="heading-2 text-color margin-24">I'm a software Developer</h2>
+       <p class="section-par text-color margin-24">I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+    see and have a project you need coded,
+    don’t hestiate to contact me..</p>
+       <div class="social-media">
+            <img  class="github" src="socialmedia/github.png" alt="github">
```

Comment:
- [Optional] Kindly link each social icon using (`a` tag) for better user interaction.